variables.tf

```diff
@@ -49,7 +49,7 @@ variable "aks_config" {
     max_nodes    = number
   })
   default = {
-    vm_size      = "Standard_D4s_v3"
+    vm_size      = "Standard_D8s_v3"
```

Comment:
Bumping the size a bit so we have 32GB nodes instead of 16GB to give some extra room.

Also at the moment in the scratch Azure account we have a limit of 1 node per cluster, so the 16GB on will not work as we can't scale up:

> The maximum node count you have selected is higher than the remaining quota for this subscription. Autoscaling may fail above 1 nodes due to quota restrictions (2 cores remaining).